x-pack/libbeat/management/manager.go

```diff
@@ -45,7 +47,7 @@ type ConfigManager struct {
 // NewConfigManager returns a X-Pack Beats Central Management manager
 func NewConfigManager(config *common.Config, registry *reload.Registry, beatUUID uuid.UUID) (management.ConfigManager, error) {
 	c := defaultConfig()
-	if config != nil {
+	if config.Enabled() && config != nil {
```

Comment:
nit: It is enough to write `if config.Enabled() {` as `common.Config` returns `false` if the struct is `nil`.

Reply:
good catch I've made the change.